source/fundamentals/crud/write-operations/bulk-write.txt

```diff
@@ -0,0 +1,552 @@
+     - | The database and collection to insert the BSON document into.
+       |
+       | **Data Type:** {+string-data-type+} or `CollectionNamespace <{+new-api-root+}/MongoDB.Driver/MongoDB.Driver.CollectionNamespace.html>`__
+
+   * - ``filter``
+     - | The **query filter** that specifies the criteria used to match documents in your collection.
+         The ``DeleteOne`` operation deletes *only the first document* that matches the
+         query filter.
+       |
+       | **Data Type:** `FilterDefinition<TDocument> <{+new-api-root+}/MongoDB.Driver/MongoDB.Driver.FilterDefinition-1.html>`__
+
+   * - ``collation``
+     - | *Optional.* The language collation to use when sorting results. See
+         the :manual:`{+mdb-server+} manual</reference/command/delete/#std-label-deletes-array-collation>`
+         for more information.
+       |
+       | **Data Type:** `Collation <{+new-api-root+}/MongoDB.Driver/MongoDB.Driver.Collation.html>`__
+       | **Default:** ``null``
+   
+   * - ``hint``
+     - | *Optional.* The index to use to scan for documents. 
+         See the :manual:`{+mdb-server+} manual</reference/command/update/#std-label-update-command-hint>`
+         for more information.
+       |
+       | **Data Type:** `BsonValue <{+new-api-root+}/MongoDB.Bson/MongoDB.Bson.BsonValue.html>`__
+       | **Default:** ``null``
+
+The following example creates an instance of the ``BulkWriteDeleteOneModel<TDocument>``
+class, and uses ``BsonDocument`` as the type for ``TDocument``. The
+filter matches the document in the ``sample_restaurants.restaurants`` collection
+where the value of the ``restaurant_id`` field is ``"5678"``.
+
+.. literalinclude:: /includes/fundamentals/code-examples/BulkWrite.cs
+   :start-after: start-bulk-delete-one
+   :end-before: end-bulk-delete-one
+   :language: csharp
+   :copyable:
+   :dedent: 8
+
+To delete multiple documents, create an instance of the ``BulkWriteDeleteManyModel<TDocument>``
+class and pass a query filter that specifies the document that you want to delete.
+The ``DeleteMany`` operation removes *all documents* that match your
+query filter.
+
+The following example creates an instance of the ``BulkWriteDeleteManyModel<TDocument>``
+class, and uses ``BsonDocument`` as the type for ``TDocument``. The
+filter matches the document in the ``sample_restaurants.restaurants`` collection
+where the value of the ``name`` field is ``"Mongo's Deli"``.
+
+.. literalinclude:: /includes/fundamentals/code-examples/BulkWrite.cs
+   :start-after: start-bulk-delete-many
+   :end-before: end-bulk-delete-many
+   :language: csharp
+   :copyable:
+   :dedent: 8
+
+Run the Write Operations
+------------------------
+
+After you define a ``BulkWriteModel`` instance for each operation that you want to perform,
+pass an ``IReadOnlyList`` collection of these instances to the ``BulkWrite()`` or
+``BulkWriteAsync()`` method. By default, these methods run the operations in the order
+they're defined in the collection.
```

Comment:
S: avoid using "collection" here to avoid ambiguity
```suggestion
After you define a ``BulkWriteModel`` instance for each operation that you want to perform,
create an instance of a type that implements ``IReadOnlyList`` to store your models.
Then, pass the list to the ``BulkWrite()`` or
``BulkWriteAsync()`` method. By default, these methods run the operations in the order
they're defined in the list.
```

Reply:
tricky ambiguity with "collection", and "list" is its own C# type. tried to reword